scripts/check-licenses.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# License check script for DNS-MAX project
+# This script checks licenses for all third-party dependencies
+
+echo "Checking licenses for DNS-MAX project..."
+
+# Change to project directory
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)